geopandas/_vectorized.py

```diff
@@ -0,0 +1,853 @@
+"""
+Compatibility shim for the vectorized geometry operations.
+
+Uses PyGEOS if available/set, otherwise loops through Shapely geometries.
+
+"""
+import warnings
+
+import numpy as np
+
+import shapely.geometry
+import shapely.wkb
+import shapely.wkt
+
+from shapely.geometry.base import BaseGeometry
+
+from . import _compat as compat
+
+try:
+    import pygeos
+except ImportError:
+    geos = None
```

Comment:
This is not needed I assume as it is in `compat`.

Reply:
It's still needed to have the `pygeos` module object to use below. But we know that we will only use it when it is not None.

Reply:
Oh, sure. Silly comment.